docs/guides/developer-guide.rst

```diff
@@ -984,14 +984,12 @@ Additional uplift notes
     might work better.
 
 
-..  |diagramFeaturesGoingIntoMasterBranch| image:: ../_static/developer-guide/diagram-features-going-into-master-branch.png
-    :width: 242.13px
-    :height: 296.50px
+.. |diagramFeaturesGoingIntoMasterBranch| image:: ../_static/developer-guide/diagram-features-going-into-master-branch.png
+   :width: 242.13px
+   :height: 296.50px
 
-..  |diagramMasterAndFeatureBranches| image:: ../_static/developer-guide/diagram-master-and-feature-branches.png
-    :width: 382.50px
-    :height: 382.50px
+.. |diagramMasterAndFeatureBranches| image:: ../_static/developer-guide/diagram-master-and-feature-branches.png
```

Comment:
@obrienben Just a note on formatting: I intentionally tab out to 4 places for all indented pieces (things like bulleted lists and in this case references to images). This makes it easier to format when information crosses multiple lines (pressing the tab key instead of having to accurately use the space bar). It's something that I'll do when I make the formatting consistent across all documents, but we can discuss the reasoning in person.